src/overlays/actors/ovl_En_Torch2/z_en_torch2.c

```diff
@@ -17,10 +17,10 @@ void EnTorch2_Draw(Actor* thisx, GlobalContext* globalCtx);
 void EnTorch2_UpdateIdle(Actor* thisx, GlobalContext* globalCtx);
 void EnTorch2_UpdateDeath(Actor* thisx, GlobalContext* globalCtx);
 
-extern Gfx D_0401C430;
-extern Gfx D_04048DF0;
-extern Gfx D_04057B10;
-extern Gfx D_04089070;
+extern Gfx object_torch2_shell_human[];
+extern Gfx object_torch2_shell_goron[];
+extern Gfx object_torch2_shell_deku[];
+extern Gfx object_torch2_shell_zora[];
```

Comment:
Is camel case used in OOT Decomp for object names?

I was using "object_<name>_<additional info>" because that's what @Rozelette did for two actors over a year ago
https://github.com/isghj5/mm/blame/7cf328f3b058c2f77da0e87334993e33032c4e56/linker_scripts/object_script.txt#L34-L38

I thought these names were matching the names found in the debug rom:
https://docs.google.com/spreadsheets/d/1J-4OwmZzOKEv2hZ7wrygOpMm0YcRnephEo3Q2FooF6E/edit#gid=1112088070

(Although I realized this _after_ I used "object_ginko" incorrectly and I need to go back and change the name to the correct object name)

Reply:
For the asset variable name we use camel case in OoT, yes. We only use snake_case for extracted textures filenames.
Since MM isn't doing objects yet, I guess it doesn't really matter.

Reply:
Leave the object segment symbols as addresses until we start to reconstruct object data. Names will come with that.